### PR TITLE
Updated Milestone3 Reflection.

### PR DESCRIPTION
**Pull request by rainbowxyt0305**

### CODE REVIEW

File: doc/reflection-milestone3.md [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+
+* instead of `dictionaries`, R uses `named lists`;
+
+* instead of `@app.callback()`, R uses `app$callback()`.
+
+It has definitely been a learning experience for us to try to switch the dashboard coding style in the R way.
+
+### 4. Reflection on some feedback we received and how it has impacted our dashboard
+
+One of the feedback we received from Joel is to tweak the stylistic layout of our histograms, so that it is more aesthetic appealing to the eyes and less color-clustered. After carefully discussing about this suggestion among the team, and assessing whether this change would align with our strategic goal of our dashboard, we implemented the change! The change in the color layout and the design of our histograms has been reflected on our current Milestone 3 dashboard app.
```
Comment (elabandari) [inline doc/reflection-milestone3.md]: A piece of feedback we received from Joel was to .....

Comment (elabandari) [reply doc/reflection-milestone3.md]: ....After carefully discussing **this** suggestion...

**Review by elabandari — CHANGES_REQUESTED** [review]

**Review by elina-linglin — COMMENTED** [review]
Looks good to me overall. For part 4, I would suggest briefly describe the layout change that we decide to make. My suggestion would be something like this: "After carefully discussing about this suggestion among the team, and assessing whether this change would align with our strategic goal of our dashboard, we implemented the change by showing the outline of the histograms instead."

**Review by elina-linglin — APPROVED** [review]
Good job Rachel!

**Review by elina-linglin — APPROVED** [review]

File: doc/reflection-milestone3.md (hunk 1 above)
Comment (rainbowxyt0305) [reply doc/reflection-milestone3.md]: Done.

**Pull request merged**